pkg/pillar/cmd/zedrouter/pbr.go

```diff
@@ -72,27 +73,30 @@ func PbrRouteAddDefault(bridgeName string, port string) error {
 			ifindex, index)
 		ifindex = index
 	}
-	MyTable := FreeTable + ifindex
-	myrt := *rt
-	myrt.Table = MyTable
-	// Clear any RTNH_F_LINKDOWN etc flags since add doesn't like them
-	if rt.Flags != 0 {
-		myrt.Flags = 0
-	}
-	log.Infof("PbrRouteAddDefault(%s, %s) adding %v\n",
-		bridgeName, port, myrt)
-	if err := netlink.RouteAdd(&myrt); err != nil {
-		errStr := fmt.Sprintf("Failed to add %v to %d: %s",
-			myrt, myrt.Table, err)
-		log.Errorln(errStr)
-		return errors.New(errStr)
+	for _, rt := range routes {
+		MyTable := FreeTable + ifindex
```

Comment:
A nit. I guess this line can be outside loop.

Reply:
Fixed